CHANGELOG.rst

```diff
@@ -1,9 +1,17 @@
 =======================================
-Infra.Ee_Utilities Release Notes
+Redhat\_Cop.Ee\_Utilities Release Notes
```

Comment:
Need to work out where this came from

Reply:
same with the last release that hasn't been merged either